app/views/devise/sessions/new.html.erb

```diff
@@ -1,35 +1,34 @@
-<%#= stylesheet_link_tag "devise" %>
+<div class="login-pg">
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name), html: { class: "login-reg" }) do |f| %>
+    <h2 class="text-2xl text-gray-100">SIGN IN</h2>
 
-<div class="devise-login">
-  <h2>Log in</h2>
-
-  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-    <div class="field">
-      <%= f.email_field :email,
-                    autofocus: true,
-                    autocomplete: "email",
-                    placeholder: "Email",
-                    class: "devise-input" %>
-    </div>
+    <%# <div class="form-logins"> %>
```

Comment:
Kindly remove this unused code for it makes your code base look untidy and may cause you to have errors.